src/digitalmarketplace/components/keyword-search/keyword-search.yaml

```diff
@@ -0,0 +1,41 @@
+params:
+- name: id
+  type: string
+  required: true
+  description: The id of the keyword search component
+- name: aria_label
+  type: string
+  description: The aria label for the keyword search component
+- name: input_id
+  type: string
+  required: true
+  description: The id of the search input
+- name: input_value
+  type: string
+  description: The value of the search input
+- name: input_label
+  type: string
+  required: true
+  description: The label of the search input
+- name: input_aria_controls
+  type: string
+  description: Aria controls for search input
```

Comment:
Can we use camelCase instead of snake_case to be consistent with govuk-frontend?

Reply:
Ah, whoops - I got caught out when I copied the old List Input code and have been snake_casing ever since. I'm going to open up a separate PR to fix that, as well as fix it on Briefs and Buyer.